### ISSUE TITLE
Persona: Narrator should read out status. And should place focus on the Persona itse

### ISSUE THREAD

**Issue by aneeshack4** [microsoft/fluentui#10851]
### Environment Information

- **Package version(s)**: (fill this out)
- **Browser and OS versions**: (fill this out)

> Note that our screen reader support is limited to Edge + Narrator. Issues that do not reproduce in Edge + Narrator, and aren't caused by obvious invalid aria values, should be filed with the respective screen reading software, not the Fabric repo.

### Describe the issue:

Bug 2 of #7643 Only colors are used to show the Persona's status and it's not interactable with Narrator.

### Please provide a reproduction of the issue in a codepen:
![image](https://user-images.githubusercontent.com/4161791/66877299-eea3a600-ef82-11e9-9af3-dffc13b0eaf9.png) on the Persona page on the Fabric website.

#### Actual behavior:

Screen reader users will miss the information of corresponding  Persona's status (like busy, available, etc) as it is not given in text form that's why screen reader doesn't announce it. It's weird that with Narrator, we can't even place focus on the actual Persona (the face).

#### Expected behavior:

Needs to read out status - provide that in text form but also maybe make it actually interactable with screenreaders first which might take care of the former.

**Comment by micahgodbolt** [comment]
we're waiting for feedback from LPC team regarding the proper approach

**Comment by xugao** [comment]
next step: try using `presenceTitle`

**Comment by xugao** [comment]
Persona is, currently by design, not a button/focus-able. since there is nothing preventing the user to wrap this piece of content and make the wrapped element accessible, i am going to downgrade the priority for now. if anything, we should provide guidance in our doc site on how to use the component in an accessible way
